code/game/machinery/doors/door.dm

```diff
@@ -349,11 +349,7 @@
 /obj/machinery/door/proc/crush()
 	for(var/mob/living/L in get_turf(src))
 		L.visible_message("<span class='warning'>[src] closes on [L], crushing [L.p_them()]!</span>", "<span class='userdanger'>[src] closes on you and crushes you!</span>")
-		if(iscarbon(L))
-			var/mob/living/carbon/C = L
-			for(var/i in C.all_wounds) // should probably replace with signal
-				var/datum/wound/W = i
-				W.crush(DOOR_CRUSH_DAMAGE)
+		SEND_SIGNAL(L, COMSIG_LIVING_DOORCRUSH)
```

Comment:
```suggestion
		SEND_SIGNAL(L, COMSIG_LIVING_CRUSHED_BY_DOOR, src)
```
* Be clear in the signal name who's the agent and who's the target of the action.
* Pass relevant information, such as the source (and the proc input args whenever they exist).